faucet/src/http.rs

```diff
@@ -19,7 +19,10 @@ use tokio::time::timeout;
 const HEALTH_DURATION: Duration = Duration::from_millis(5000);
 
 const KV_STORE_NAME: &str = "store";
-const FAUCET_COOLDOWN_HOURS: i64 = 6;
+const FAUCET_COOLDOWN_HOURS: i64 = 12;
```

Comment:
why?

Reply:
Yea but there have been a lot of instances where something went wrong, and users had to wait for the cooldown before they got the vault up and running